<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vaanhit" name="Default test" tests="2" failures="0" timestamp="7 Oct 2016 07:02:58 GMT" time="3.411" errors="0">
  <testcase name="@BeforeClass setup" time="0.0" classname="com.selenium.DemoTMTestNG">
    <failure type="java.io.FileNotFoundException" message="C:\WORK_ENV\workspace\demoTMApplicationSelenium\src\test\java\com\testData\TestData.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\WORK_ENV\workspace\demoTMApplicationSelenium\src\test\java\com\testData\TestData.xlsx (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at com.selenium.utility.ExcelUtils.setExcelFile(ExcelUtils.java:33)
	at com.selenium.DemoTMTestNG.setup(DemoTMTestNG.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase name="@AfterClass exit" time="0.0" classname="com.selenium.DemoTMTestNG">
    <skipped/>
  </testcase> <!-- @AfterClass exit -->
  <testcase name="LoginTestCase1" time="0.001" classname="com.selenium.DemoTMTestNG">
    <skipped/>
  </testcase> <!-- LoginTestCase1 -->
  <testcase name="headerCheckTestCase1" time="0.001" classname="com.selenium.DemoTMTestNG">
    <skipped/>
  </testcase> <!-- headerCheckTestCase1 -->
</testsuite> <!-- Default test -->
